MapboxDirections/MBRouteStep.swift

```diff
@@ -506,10 +506,17 @@ public class RouteStep: NSObject {
      */
     public let transportType: TransportType?
     
+    
+    /**
+     Destinations signs for the current way.
```

Comment:
The documentation is written for an audience that may not be familiar with routers. How about something like this:

> Destinations, such as [control cities](https://en.wikipedia.org/wiki/Control_city), that appear on guide signage for the road identified in the `name` property.
> 
> This property is typically available in steps leading to or from a freeway or expressway.